src/components/CustomInput.vue

```diff
@@ -8,6 +8,7 @@
       :name="name"
       @change="selectedAnswer"
     > 
+    <img v-if="image" :class="{'custom-input__image' :image}" :src="image" alt="">
```

Comment:
Really tiny thing here, but a linter will shout at you in future for this, so get into the habit of defining objects with the space after the colon rather than before it 😄 

```suggestion
    <img v-if="image" :class="{'custom-input__image': image}" :src="image" alt="">
```